libs/expo/betterangels/src/lib/screens/PublicNote/index.tsx

```diff
@@ -89,92 +92,94 @@ export default function PublicNote({ noteId }: { noteId: string }) {
   }
 
   return (
-    <MainScrollContainer pb={40} bg={Colors.NEUTRAL_EXTRA_LIGHT}>
+    <>
+      <MainScrollContainer pb={40} bg={Colors.NEUTRAL_EXTRA_LIGHT}>
+        <View
+          style={{
+            gap: Spacings.sm,
+            flex: 1,
+          }}
+        >
+          <TextBold size="lg">Write Public Note</TextBold>
+          {autoNote !== publicNote && (
+            <View
+              style={{
+                padding: Spacings.xs,
+                borderRadius: 8,
```

Comment:
```suggestion
                borderRadius: Spacings.xs,
```

Reply:
Thanks for catching it. I just created a subtask that will fix them at the application level. [Refactor the application to use Constant values instead of Hard coded values](https://betterangels.atlassian.net/browse/DEV-536)